editor/editor_log.cpp

```diff
@@ -71,6 +71,14 @@ void EditorLog::_notification(int p_what) {
 			}
 		}
 	}
+
+	if (p_what == NOTIFICATION_ENTER_TREE || p_what == NOTIFICATION_THEME_CHANGED) {
```

Comment:
Since we're touching this methods, I'd prefer if we cleaned it up, used switch, and unified the code above. I see no reason why it's split up there to begin with.

Reply:
I wanted to use switch at first, but there is some confusing branching (ENTER_TREE and THEME_CHANGED cases are not the same), so I decided not to touch it >_>

Reply:
Given that the theme changed notification fires after enter tree anyway, I think we can just take its code as the relevant one. Something like for both notifications:

```cpp
if (log != nullptr) {
	Ref<DynamicFont> df_output_code = get_font("output_source", "EditorFonts");
	if (df_output_code.is_valid()) {
		log->add_font_override("normal_font", get_font("output_source", "EditorFonts"));
		log->add_color_override("selection_color", get_color("accent_color", "Editor") * Color(1, 1, 1, 0.4));
	}
}
```